From d9f35b542adbf6b0e1114c7c077df04212a98fc7 Mon Sep 17 00:00:00 2001
From: Jean Felder <jfelder@src.gnome.org>
Date: Thu, 18 Nov 2021 12:09:26 +0100
Subject: [PATCH] meson: Remove incorrect i18n.merge_file argument

The positional argument was being silently ignored until meson 0.60.0
where it returns a deprecation message:
"DEPRECATION: i18n.merge_file does not take any positional
arguments. This will become a hard error in the next Meson release."

See: https://github.com/mesonbuild/meson/issues/9441
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 25ff1e102..145ee5221 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -47,7 +47,6 @@ appdata_conf.set('appid', APPLICATION_ID)
 appdata_conf.set('package_url', PACKAGE_URL)
 appdata_conf.set('package_url_bug', PACKAGE_URL_BUG)
 i18n.merge_file(
-  'appdata',
   input: configure_file(
     output: PROJECT_RDNN_NAME + '.appdata.xml.in',
     input: PROJECT_RDNN_NAME + '.appdata.xml.in.in',
-- 
GitLab

